Authentication/Login/View/LoginConfiguration.swift

```diff
@@ -13,7 +13,7 @@ import Foundation
 */
 public protocol LoginControllerTransitionDelegate {
```

Comment:
In the protocol `SessionServiceType` which the user must always implement, we ask that they have a property `var events: Signal<SessionServiceEvent<User>, NoError> { get }`, where 
```swift
public enum SessionServiceEvent<User: UserType> {
    case LogIn(User)
    case LogOut(User)
    case SignUp(User)
    case LogInError(SessionServiceError)
    case SignUpError(SessionServiceError)
}
```
And the only place we use this signal is in the `AuthentiationBootstrapper`:
```swift
sessionService.events.observeNext { [unowned self] event in
            switch event {
            case .LogIn(_): self._window.rootViewController = self._mainViewControllerFactory()
            case .LogOut(_): self._window.rootViewController = UINavigationController(rootViewController: self.createLoginController())
            default: break
            }
        }
```

And if we now add `userDidLog` in the `LogInTransitionDelegate` protocol, we wouldn't even need the `LogIn`event.

The user when seeing that he must conform to that protocol, I think he/she will think he must send all that events, when we don't really need those event notifications from him/her.